bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -2492,7 +2492,7 @@ components:
           description: |
             A list of `References`
```

Comment:
It would have been easy to trace if you could provide the description as "A list of Shipper References